networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -82,3 +84,13 @@ def test_to_vertex_cover(self):
         matching = maximum_matching(self.graph)
         vertex_cover = to_vertex_cover(self.graph, matching)
         self.check_vertex_cover(vertex_cover)
+
+
+def test_eppstein_matching():
+    # Special test
```

Comment:
I've been having trouble with docstrings in the test_* methods. nosetests uses the docstring to identify which test run/fail/error.  It is much more useful (to me) to report the file and name of the test rather than a docstring. Could we avoid docstrings on test methods until we figure out a way to make it work well with nose?

Is there another way to put comments? We don't create documentation for the tests, so maybe ```#``` comments are not a bad way to go. We could also use triple quotes just like for docstrings, but put them just before the declaration line instead of after.

    """test that Eppstein and max_cardinality agree"""
    def test_eppstein_matching():
        G = nx.Graph()

What do people think?